cmd/ui/src/views/Explore/GraphView.tsx

```diff
@@ -233,7 +232,7 @@ const GraphView: FC = () => {
                 <ExploreSearch />
                 <GraphControls
                     layoutOptions={baseGraphLayouts}
-                    isExploreTableSelected={isExploreTableSelected}
+                    isExploreTableSelected={isExploreTableSelected ?? false}
```

Comment:
this was temporary so TS would be happy, we may need to figure out a better way to handle this!